src/neural/onnx/builder.cc

```diff
@@ -135,12 +135,15 @@ std::string PopulateStdNodeFields(pblczero::NodeProto* node,
 std::string OnnxBuilder::Conv(const std::string& name,
                               const std::string& input_name,
                               const OnnxConst& kernel_weights,
-                              const OnnxConst& bias_weights, int pads) {
+                              const OnnxConst& bias_weights, 
+                              int filters,
```

Comment:
Why not just do a 'kernel_weights.GetDimensions().back()` instead of passing the number of filters?

Reply:
Agreed, that's better